app/models/npq_validation_data.rb

```diff
@@ -4,6 +4,8 @@ class NPQValidationData < ApplicationRecord
   # TODO: Rename table
   self.table_name = "npq_profiles"
 
+  has_paper_trail
```

Comment:
We don't want this papertrailed. This contains all the PII that we're going to delete

Reply:
can we set those as a dependency so when the validation data is deleted so it the paper trail versions? my understanding at the moment is that `NPQValidationData` is the "profile" as we not sorted this part out yet

Reply:
I've pulled the commit out. I'll move part of it into another PR once I'm done here